root/static/scripts/guess-case/MB/GuessCase/Handler/Artist.js

```diff
@@ -165,6 +165,8 @@ MB.GuessCase.Handler.Artist = function (gc) {
 
                 return utils.trim(_.compact(names).join(" ") + (append || ""));
             }
+
+            return null;
```

Comment:
I believe the result of this function is string-joined by `sortCompoundName`.  It's true that `String.prototype.join` converts `null` to `''` when joining, but I think `return '';` would be a bit clearer.